### PR TITLE
testing/lazperf: new aport

### PR DESCRIPTION
**Pull request by chambbj**
https://github.com/hobu/laz-perf
Alternative LAZ implementation for C++ and JavaScript

### CODE REVIEW

File: testing/lazperf/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Contributor: Bradley J Chambers <brad.chambers@gmail.com>
+# Maintainer: Bradley J Chambers <brad.chambers@gmail.com>
+pkgname=laz-perf
+pkgver=1.1.0
+pkgrel=0
+pkgdesc="Alternative LAZ implementation for C++ and JavaScript"
+url="https://github.com/hobu/laz-perf"
+arch="all"
+license="LGPL2+"
+depends="libgcc libstdc++ musl"
```
Comment (ncopa) [inline testing/lazperf/APKBUILD]: Those are not needed. abuild will autodetect them.

File: testing/lazperf/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+makedepends="cmake"
+install=""
+subpackages="$pkgname-dev"
+source="$pkgname-$pkgver.tar.gz::https://github.com/hobu/$pkgname/archive/$pkgver.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver"
+
+build() {
+        mkdir -p "$builddir"
```
Comment (ncopa) [inline testing/lazperf/APKBUILD]: should not be needed to create `$builddir`

File: testing/lazperf/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+package() {
+        cd "$builddir"/build
+        make DESTDIR="$pkgdir" install
+}
+
+prepare() {
+        default_prepare
+}
```
Comment (ncopa) [inline testing/lazperf/APKBUILD]: this prepare() function should be removed

**Comment by chambbj** [comment]
As requested @ncopa

**Pull request closed without merging**